x/claims/keeper/claim.go

```diff
@@ -158,26 +159,25 @@ func (k Keeper) MergeClaimsRecords(
 
 	// iterate over all the available actions and claim the amount if
 	// the recipient or sender has completed an action but the other hasn't
-	for i := int32(1); i < int32(len(types.Action_value)); i++ {
-		senderCompleted := senderClaimsRecord.ActionsCompleted[i-1]
-		recipientCompleted := recipientClaimsRecord.ActionsCompleted[i-1]
 
-		action := types.Action(i)
+	for _, action := range actions {
```

Comment:
> // MergeClaimsRecords merges two claim records from the
// sender and recipient of the IBC transfer while claiming the
// amount for the all the sender actions on behalf of the recipient.

Method godoc is confusing